### ISSUE TITLE
ValueError: attribute 'Elements' is not optional for entity instance of type 'IFC4.IfcGeometricCurveSet'

### ISSUE THREAD

**Issue by blenderarc** [IfcOpenShell/IfcOpenShell#4670]
```
Python: Traceback (most recent call last):
  File "C:\Users\Sahru\AppData\Roaming\Blender Foundation\Blender\4.1\scripts\addons\blenderbim\bim\ifc.py", line 360, in execute_ifc_operator
    result = getattr(operator, "_execute")(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Sahru\AppData\Roaming\Blender Foundation\Blender\4.1\scripts\addons\blenderbim\bim\module\project\operator.py", line 1196, in _execute
    ifc_exporter.export()
  File "C:\Users\Sahru\AppData\Roaming\Blender Foundation\Blender\4.1\scripts\addons\blenderbim\bim\export_ifc.py", line 52, in export
    self.sync_edited_objects()
  File "C:\Users\Sahru\AppData\Roaming\Blender Foundation\Blender\4.1\scripts\addons\blenderbim\bim\export_ifc.py", line 128, in sync_edited_objects
    bpy.ops.bim.update_representation(obj=obj.name)
  File "C:\Program Files\Blender Foundation\Blender 4.1\4.1\scripts\modules\bpy\ops.py", line 109, in __call__
    ret = _op_call(self.idname_py(), kw)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Error: Python: Traceback (most recent call last):
  File "C:\Users\Sahru\AppData\Roaming\Blender Foundation\Blender\4.1\scripts\addons\blenderbim\libs\site\packages\ifcopenshell\entity_instance.py", line 332, in __setitem__
    self.wrapped_data.setArgumentAsNull(idx)
  File "C:\Users\Sahru\AppData\Roaming\Blender Foundation\Blender\4.1\scripts\addons\blenderbim\libs\site\packages\ifcopenshell\ifcopenshell_wrapper.py", line 5285, in setArgumentAsNull
    return _ifcopenshell_wrapper.entity_instance_setArgumentAsNull(self, i)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Attribute not set

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Sahru\AppData\Roaming\Blender Foundation\Blender\4.1\scripts\addons\blenderbim\bim\ifc.py", line 360, in execute_ifc_operator
    result = getattr(operator, "_execute")(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Sahru\AppData\Roaming\Blender Foundation\Blender\4.1\scripts\addons\blenderbim\bim\module\geometry\operator.py", line 324, in _execute
    self.update_obj_mesh_representation(context, obj)
  File "C:\Users\Sahru\AppData\Roaming\Blender Foundation\Blender\4.1\scripts\addons\blenderbim\bim\module\geometry\operator.py", line 403, in update_obj_mesh_representation
    new_representation = ifcopenshell.api.run("geometry.add_representation", self.file, **representation_data)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Sahru\AppData\Roaming\Blender Foundation\Blender\4.1\scripts\addons\blenderbim\libs\site\packages\ifcopenshell\api\__init__.py", line 166, in run
    return usecase_function(ifc_file, should_run_listeners=should_run_listeners, **settings)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Sahru\AppData\Roaming\Blender Foundation\Blender\4.1\scripts\addons\blenderbim\libs\site\packages\ifcopenshell\api\__init__.py", line 326, in wrapper
    result = usecase(*args, **settings)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Sahru\AppData\Roaming\Blender Foundation\Blender\4.1\scripts\addons\blenderbim\libs\site\packages\ifcopenshell\api\geometry\add_representation.py", line 64, in add_representation
    return usecase.execute()
           ^^^^^^^^^^^^^^^^^
  File "C:\Users\Sahru\AppData\Roaming\Blender Foundation\Blender\4.1\scripts\addons\blenderbim\libs\site\packages\ifcopenshell\api\geometry\add_representation.py", line 80, in execute
    return self.create_plan_representation()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Sahru\AppData\Roaming\Blender Foundation\Blender\4.1\scripts\addons\blenderbim\libs\site\packages\ifcopenshell\api\geometry\add_representation.py", line 177, in create_plan_representation
    return self.create_annotation2d_representation()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Sahru\AppData\Roaming\Blender Foundation\Blender\4.1\scripts\addons\blenderbim\libs\site\packages\ifcopenshell\api\geometry\add_representation.py", line 840, in create_annotation2d_representation
    items = [self.file.createIfcGeometricCurveSet(self.create_curves(is_2d=True))]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Sahru\AppData\Roaming\Blender Foundation\Blender\4.1\scripts\addons\blenderbim\libs\site\packages\ifcopenshell\file.py", line 353, in create_entity
    e[idx] = arg
    ~^^^^^
  File "C:\Users\Sahru\AppData\Roaming\Blender Foundation\Blender\4.1\scripts\addons\blenderbim\libs\site\packages\ifcopenshell\entity_instance.py", line 335, in __setitem__
    raise ValueError(
ValueError: attribute 'Elements' is not optional for entity instance of type 'IFC4.IfcGeometricCurveSet'
Location: C:\Program Files\Blender Foundation\Blender 4.1\4.1\scripts\modules\bpy\ops.py:109
```

**Comment by Andrej730** [comment]
Need an example how to reproduce the issue.

**Comment by Andrej730** [comment]
Should be resolved after 341cdd4
